DashSync/shared/Models/Entities/DSProviderRegistrationTransactionEntity+CoreDataClass.m

```diff
@@ -77,8 +78,7 @@ - (DSTransaction *)transactionForChain:(DSChain *)chain {
         transaction.providerType = self.providerType;
         transaction.providerMode = self.providerMode;
         transaction.collateralOutpoint = self.collateralOutpoint.transactionOutpoint;
-        transaction.ipAddress = self.ipAddress.UInt128;
-        transaction.port = self.port;
+        transaction.masternodeAddress = (DSAddress){self.ipAddress.UInt128, self.port};
```

Comment:
I think this should stay as ipAddress and port, because this represents more fields than anything else: see DIP3 https://github.com/dashpay/dips/blob/master/dip-0003.md#registering-a-masternode-proregtx